### DISCUSSION TITLE
Support for calculated columns to the Data Explorer

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2274] [Feature Ideas]
Mathesar should support adding generated columns in the Data Explorer. Users would set up a formula for how the data is calculated.

Related features:

- [Allow calculated and dynamic default values](https://github.com/mathesar-foundation/mathesar/discussions/2272)
- [Support for calculated columns in tables](https://github.com/mathesar-foundation/mathesar/discussions/2271)

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by jeromeperez** [comment]
Hi
Good idea, it should be very interesting
exemple:
table taxa fields totalfragment, taxa
table sample fields fragment, sample
table sample_taxa fields taxa,sample
totalfragment = Sum(fragment) with sample_taxa.taxa = taxa and sample_taxa.sample = sample
Creating an exploration for the three table grouping by sample/taxa and fill the field totalfragment with the calculation
